src/features/message-center/components/message-center-modal/MessageCenterModal.js

```diff
@@ -227,7 +242,6 @@ function MessageCenterModal({
             className="bdl-MessageCenterModal"
             data-resin-component="messageCenterModal"
             data-testid="messagecentermodal"
-            focusElementSelector=".bdl-CategorySelector .is-selected"
```

Comment:
Will removing this affect the focused element for when there are categories?

Reply:
yes it will, but it needs to be removed as the categories will now automatically show anymore and an error is thrown as it doing a querySelector on this on mount